### PR TITLE
Adds notice regarding watch face development changes

### PR DESCRIPTION
**Pull request by garanj**
Adds notice regarding watch face development changes

### CODE REVIEW

File: Complications/README.md [hunk 1]
```diff
@@ -5,6 +5,11 @@ Android Complications Sample
 Complication Test Suite is a set of complication data sources that provide dummy data and it can be
 used to test how different types of complications render on a watch face.
 
+> [!IMPORTANT]  
+> As of July 10, 2024, watch faces must use the [Watch Face Format][1] ([sample][3]) in order to be installed on new watches that launch with Wear OS 5 pre-installed. Learn more about the user-facing changes in this [Help Center][2] article.
```
Comment (khufdev) [inline Complications/README.md:9]: Incorrect recycling of reference links (which we use at the end of the introduction)

**Review by khufdev — COMMENTED** [review]
Found one instance of reference links sending readers to the incorrect page

File: WatchFaceKotlin/README.md [hunk 2]
```diff
@@ -9,8 +9,9 @@ Demonstrates watch faces using the androidX libraries (Kotlin).
 
 Introduction
 ------------
-The AndroidX watch face libraries allow you to develop a watch face service in Kotlin. However,
-[WFF][1] is strongly recommended in preference to this approach.
+The AndroidX watch face libraries allow you to develop a watch face service in Kotlin.
```
Comment (khufdev) [inline WatchFaceKotlin/README.md]: We'd want to keep the recommendation to use Watch Face Format, right?

File: Complications/README.md (hunk 1 above)
Comment (garanj) [reply Complications/README.md]: I think I've fixed this!

File: WatchFaceKotlin/README.md (hunk 2 above)
Comment (garanj) [reply WatchFaceKotlin/README.md]: Thanks, I think I've fixed this!

**Comment by LouisCAD** [comment]
There are no nice words for what's happening.

**Pull request merged**